behaviors/TranslatableModel.php

```diff
@@ -208,13 +227,11 @@ protected function loadTranslatableData($locale = null)
             return $this->translatableAttributes[$locale] = [];
         }
 
-        $obj = Db::table('rainlab_translate_attributes')
-            ->where('locale', $locale)
-            ->where('model_id', $this->model->getKey())
-            ->where('model_type', get_class($this->model))
-            ->first();
+        $obj = $this->model->translations->first(function ($value, $key) use($locale) {
+            return $value->attributes['locale'] === $locale;
+        });
 
-        $result = $obj ? json_decode($obj->attribute_data, true) : [];
+        $result = $obj ? json_decode($obj['attribute_data'], true) : [];
```

Comment:
@massimomegistus Would you mind letting us know the reason for changing this line? The original line should've worked.

Reply:
@bennothommo there's no reason, In fact. That's a useless change.

Reply:
@massimomegistus Feel free to restore it for us :)